src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+    public Task(String description) {
+        setDescription(description);
+        setDone(false);
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isDone() {
```

Comment:
Perhaps the method could sound more verb-like?

```suggestion
    public boolean checkIsDone() {
```